docs/resource-policy/policy/balloons.md

```diff
@@ -86,26 +86,33 @@ Balloons policy parameters:
   balloons. If `true`, new balloons are created using CPUs on
   NUMA/die/package with most free CPUs, that is, balloons are spread
   across the hardware topology. This helps inflating balloons within
-  the same NUMA/die/package and reduces interference between workloads
+  the same NUMA/die/package and reduces interference between containers
   in balloons when system is not fully loaded. The default is `false`:
   pack new balloons tightly into the same NUMAs/dies/packages. This
   helps keeping large portions of hardware idle and entering into deep
   power saving states.
 - `preferSpreadOnPhysicalCores` prefers allocating logical CPUs
   (possibly hyperthreads) for a balloon from separate physical CPU
-  cores. This prevents workloads in the balloon from interfering with
+  cores. This prevents containers in the balloon from interfering with
   themselves as they do not compete on the resources of the same CPU
   cores. On the other hand, it allows more interference between
-  workloads in different balloons. The default is `false`: balloons
+  containers in different balloons. The default is `false`: balloons
   are packed tightly to a minimum number of physical CPU cores. The
   value set here is the default for all balloon types, but it can be
   overridden with the balloon type specific setting with the same
   name.
 - `balloonTypes` is a list of balloon type definitions. The order of
-  the types is significant in balloon CPU allocation and in choosing a
-  balloon type for a new container. The first type with the highest
-  `allocatorPriority` will get CPUs before other balloons when their
-  `minBalloons` > 0. A new container without balloon annotations will
+  the types is significant in two cases.
+
+  In the first case the policy pre-creates `minBalloons` balloons from
```

Comment:
Could/should we explain what "minBalloons balloons" means? Or is it covered somewhere else in the doc?

Reply:
Thanks @marquiz! I was thinking about this, too. I added now a forward-reference here to associate pre-creation (and CPU pre-allocation) to two parameters.

I originally considered also explaining the semantics for the list order after the parameters in order to avoid forward-reference, but the list of parameters is pretty long, so the end result didn't really feel better.

Maybe this version is easier to understand. I hope you already forgot everything you read previously in the doc, so you can review it again as a new user. :D